sdk-base/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPlugin.kt

```diff
@@ -140,18 +140,22 @@ interface CameraAnimationsPlugin : MapPlugin {
   ): ValueAnimator
 
   /**
-   * Create CameraBearingAnimator. Current map camera option will be applied on animation start.
+   * Create CameraBearingAnimator. Current map camera option will be applied on animation start if not specified explicitly with [options.startValue].
    *
    * @param options animator options object to set targets and other non mandatory options
+   * @param optimizePath if set to True shortest bearing path will be applied while animating bearing values.
+   * If set to False clock-wise rotation will be used if next target is greater or equal than current one
+   * and counter clock-wise rotation will be used if next target less than current one.
    * @param block optional block to apply any [ValueAnimator] parameters
    */
   fun createBearingAnimator(
     options: CameraAnimatorOptions<Double>,
+    optimizePath: Boolean = true,
```

Comment:
I wish Swift allowed us to set default values in interface (protocol) definitions.

Reply:
@macdrevx you could always switch to [Kotlin](https://kotlinlang.org/lp/mobile/) for developing iOS as well 😉